examples/with-angular-thirdpartyemailpassword/src/app/home/home.component.ts

```diff
@@ -0,0 +1,37 @@
+import { AfterViewInit, Component, OnChanges, OnDestroy, SimpleChanges } from "@angular/core";
+
+import * as Session from "supertokens-auth-react/recipe/session";
+import { signOut } from "supertokens-auth-react/recipe/thirdpartyemailpassword";
```

Comment:
This is an application route.. ideally it should not use supertokens-auth-react and only use supertokens-website.